README.md

```diff
@@ -76,7 +76,8 @@ Parameters are provided to configure the behavior of the bridge. These parameter
   * __client_topic_whitelist__: List of regular expressions ([ECMAScript grammar](https://en.cppreference.com/w/cpp/regex/ecmascript)) of whitelisted client-published topic names. Defaults to `[".*"]`.
  * __send_buffer_limit__: Connection send buffer limit in bytes. Messages will be dropped when a connection's send buffer reaches this limit to avoid a queue of outdated messages building up. Defaults to `10000000` (10 MB).
  * __use_compression__: Use websocket compression (permessage-deflate). Suited for connections with smaller bandwith, at the cost of additional CPU load.
- * __capabilities__: List of supported [server capabilities](https://github.com/foxglove/ws-protocol/blob/main/docs/spec.md). Defaults to `[clientPublish,parameters,parametersSubscribe,services,connectionGraph]`.
+ * __capabilities__: List of supported [server capabilities](https://github.com/foxglove/ws-protocol/blob/main/docs/spec.md). Defaults to `[clientPublish,parameters,parametersSubscribe,services,connectionGraph,assets]`.
+ * __asset_uri_whitelist__: List of regular expressions ([ECMAScript grammar](https://en.cppreference.com/w/cpp/regex/ecmascript)) of allowed asset URIs. Defaults to `["package://.*"]`.
```

Comment:
How does a user know what patterns would be valuable to put in here? There is no documentation explaining how `file://` urls work.

Reply:
`allowlist`?

https://inclusivenaming.org/word-lists/tier-1/whitelist/

Reply:
It uses [resource_retriever](https://index.ros.org/p/resource_retriever/github-ros-resource_retriever) under the hood which is able to resolve `package://` URLs but also `file://` and `http(s)://`. Will extend the documentation